### PR TITLE
fix(config): update default dependencyDashboardHeader

### LINKED ISSUE #15322: Improve `default` text for `dependencyDashboardHeader`
### Intro

The Dependency Dashboard has some `default` text which I want to change so it:

- Uses plain language
- Uses a better name for the link
- Ends the last sentence with a full stop

### Current header

> This issue provides visibility into Renovate updates and their statuses. [Learn more](https://docs.renovatebot.com/key-concepts/dashboard/)

### My proposed header

This issue lists all Renovate updates and their statuses. It also shows the dependencies that Renovate detected in your repository. Read the [Dependency Dashboard docs](https://docs.renovatebot.com/key-concepts/dashboard/) to learn more.

### Source

https://github.com/renovatebot/renovate/blob/1268dac93775b7e9e99592873d6baa7b35590e0c/lib/config/options/index.ts#L451-L458

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Update the default `dependencyDashboardHeader`
  - Use plain language
  - Use a better name for the link
  - Ends the last sentence with a full stop

## Context

Closes #15322.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/options/index.ts
Comment (rarkins) [inline lib/config/options/index.ts:462]:
```suggestion
      'This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.',
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/config/options/index.ts
Comment (HonkingGoose) [reply lib/config/options/index.ts:462]: I don't know how to fix this warning. I'll let the maintainers decide what we should do here. 😉

**Review by rarkins — APPROVED** [review]

**Comment by HonkingGoose** [comment]
We still need to fix the warning before this can be merged?

**Review by viceice — APPROVED** [review]

**Pull request merged**
